.rubocop.yml

```diff
@@ -46,3 +49,12 @@ DotPosition:
 # Allow class and message or instance raises
 Style/RaiseArgs:
   Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - spec/**/*
+
+Style/FileName:
+  Exclude:
+    - Gemfile
+    - Guardfile
```

Comment:
Usually my editor does that for me but it didn't this time for some reason - maybe something to do with it being a YML file? Fixed that now and will merge once it passes.